library/std/src/sys/stdio/unix_like.rs

```diff
@@ -89,7 +96,7 @@ impl io::Write for Stderr {
 }
 
 pub fn is_ebadf(err: &io::Error) -> bool {
-    err.raw_os_error() == Some(libc::EBADF as i32)
+    err.raw_os_error() == Some(EBADF as i32)
 }
 
 pub const STDIN_BUF_SIZE: usize = crate::sys::io::DEFAULT_BUF_SIZE;
```

Comment:
Did you mean to change Hermit from `pub const STDIN_BUF_SIZE: usize = 128;` to `DEFAULT_BUF_SIZE`?

```rust
pub const DEFAULT_BUF_SIZE: usize = if cfg!(target_os = "espidf") { 512 } else { 8 * 1024 };
```

Reply:
No, I didn't. Whoops! 😄 Though maybe that isn't too bad a thing anyway, @mkroening?

Reply:
Yeah, that sounds sensible to me, thanks! :) 👍